modules/lotamePanoramaIdSystem.js

```diff
@@ -188,15 +188,15 @@ export const lotamePanoramaIdSubmodule = {
 
       if (consentData && utils.isBoolean(consentData.gdprApplies)) {
```

Comment:
hmmm - i would have expected the unit tests to catch this so maybe an update is needed there but this entire block doesn't get entered if there is no consentData object populated.